reflex-dom-core/src/Reflex/Dom/Builder/Hydratable.hs

```diff
@@ -60,10 +62,30 @@ instance PrimMonad m => PrimMonad (HydratableT m) where
   type PrimState (HydratableT m) = PrimState m
   primitive = lift . primitive
 
+
+data Hydratable = Hydratable deriving Eq
```

Comment:
What's the benefit provided by making this a special IsAttribute instance rather than just a normal text attribute?

Reply:
Oh, nothing here. I suppose we could strip them out when patching for hydration though.